pkg/storegateway/series_refs.go

```diff
@@ -106,14 +106,14 @@ func (b seriesChunkRefsSet) release() {
 // seriesChunkRefs holds a series with a list of chunk references.
 type seriesChunkRefs struct {
 	lset   labels.Labels
-	chunks []seriesChunkRef
+	groups []seriesChunkRefsGroup
```

Comment:
[nit] I would clarify it's a group of chunk refs.

```suggestion
	chunkGroups []seriesChunkRefsGroup
```